_SQLServer/TableLoadAbilities.sql

```diff
@@ -155,36 +156,85 @@ insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityA
 
 
+--Von Drakk
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Shapeshift', 'Action', 'Support', 1, -1, 0, 'Nocturne', '1.0'); --? Check the values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Ghastly Vigor', 'Action', 'Support', 1, 0, 0, 'Augment, Aura 3, All Undead gain +1B Attack', '1.0'); --? Check the values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Tide of Bones', 'Action', 'Support', 2, -1, 0, 'Wave 2, Raise', '1.0'); --? Check the values
 
 
+--1.0
+--Miniboss
 
 
+--Captain R
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Flintlock', 'Action', 'Offense', 1, -1, 0, 'Missle 6, +1R DEX', '1.0'); --? Check the values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Deep One Attack', 'Action', 'Offense', 2, 0, 0, 'Magic, Cross 5, +1R WILL', '1.0'); --? Check the values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('X Marks the Spot', 'Action', 'Support', 2, -1, 0, 'Draw and equip 1 Loot card', '1.0'); --? Check the values
+ look at later
 
 
+--Death Spectre
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Lifesap', 'Action', 'Offense', 1, -1, 1, 'Range 1.  Do not make an attack roll.  Target model takes one wound.  Death immediately heals one wound.', '1.0'); --? Check the values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Ghost Form', 'Action', 'Support', 2, 0, 0, 'Teleport', '1.0'); --? Check the values
 
 
+--Herald of Vulcanis
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Javelin', 'Action', 'Offense', 1, -1, 6, 'Missle 6, Fire.', '1.0'); --? Check the values
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Shield Bash', 'Action', 'Offense', 2, 0, 0, '+1 Star Attack, Massive Damage', '1.0'); --? Check the values
 
 
+--Kaelly the Nether Strider
+insert into Abilities (Name, AbilityResource, AbilityType, AbilityCost, AbilityAttribute, AbilityRange, AbilityDescription, AbilityVersion) VALUES ('Corrosive Striked', 'Action', 'Offense', 1, -1, 6, 'Missle 6, Fire.', '1.0'); --? Check the values
```

Comment:
AbilityCost should be 2.
AbilityAttribute should be 1.
AbilityRange is technically 8.